src/application/config/routes.php

```diff
@@ -120,6 +120,7 @@
 $route['api/v2/categories/providers/(:any)']['get'] = 'api/v2/categoriesv2/getCategoryByProviderId/$1';
 $route['api/v3/patients/(:any)']['get'] = 'api/v3/patientsv3/getPatient/$1';
 $route['api/v3/appointments/(:any)']['get'] = 'api/v3/appointmentsv3/getUserAppointments/$1';
+$route['api/v3/appointments/service/(:any)/address-statistic']['get'] = 'api/v3/appointmentsv3/getAppointmentsAddressStatistic/$1';
```

Comment:
/service/ -> /services/

Reply:
is this API is only for district statistic, if so please change uri to

api/v3/appointments/services/(:any)/statistics/district

Reply:
updated